.woloxci/config.yml

```diff
@@ -0,0 +1,25 @@
+config:
+  dockerfile: .woloxci/Dockerfile
+  project_name: rails-bootstrap
```

Comment:
Shouldn't we change this name when we run the bootstrap script?

Reply:
That should be taken care of by the bootstrap script, which searches the entire repo for the word `rails-bootstrap`